contentcuration/contentcuration/frontend/settings/pages/UsingStudio/index.vue

```diff
@@ -141,4 +145,7 @@
   h2 {
     margin-top: 32px;
   }
-</style>
+  .kexternal-redirect{
+    margin-left: -8px;
```

Comment:
The negative margin moves the element slightly to the left of its normal position. This could cause misalignment with other elements on the page, especially if the layout is responsive or if the link appears in different contexts.

Reply:
Hey @AllanOXDi , I think issue lies in `KExternalLink` component. The component by default is adding an inline 8px left margin. That's why I added -8px to realign back. [For More](https://github.com/learningequality/kolibri-design-system/blob/d3865313e65eab24195a2b65e411506b364ec9ba/lib/buttons-and-links/KExternalLink.vue#L106)

<img width="1395" alt="image" src="https://github.com/user-attachments/assets/13a4dfee-e3c6-4c3a-809c-f5edf8bf1106">

Unfortunately, I wasn't able to find other solutions for this.

Reply:
Thanks both. @shivam-daksh I think it's fine to remove `margin-left: -8px;` from here, as @AllanOXDi suggests.

If we feel it's important, we can consider making adjustments directly to `KExternalLink` in the `kolibri-design-system` repository in the future, but for now I would just use the default behavior since it will be consistent with other links.

Reply:
Sure @MisRob, I've removed the `margin-left:-8px` . But it is causing little indendation issue in Settings->About Studio page. 

<img width="835" alt="image" src="https://github.com/user-attachments/assets/33636263-b739-42a7-bc56-860e55ae5f91">

Reply:
Hi @shivam-daksh 
 you could try this
```
.kexternal-redirect {
  margin-left: 0 ;
}
```

Reply:
Actually this won't work -sorry

Reply:
It's okay. Let me know if we have a fix for now.

Reply:
Only way I think of now is add the class in **About Studio** page only.

‘.kexternal-redirect { margin-left: 0 ; }‘

Reply:
That sounds good @shivam-daksh. You're right that it needs to be aligned better on this page, thanks for elaborating.

Reply:
So, should I do it?

Reply:
Yes

Reply:
That's done.

Reply:
Thanks!